lib/katello/plugin.rb

```diff
@@ -669,7 +669,7 @@ def katello_template_setting_values(name)
         type: :integer,
         default: 1440,
         full_name: N_('Orphaned Content Protection Time'),
-        description: N_('Time in minutes to consider orphan content as orphaned.')
+        description: N_('Time in minutes before content not contained within a repo nor recently touched will be considered as orphaned.')
```

Comment:
How about "Time in minutes before content that is not contained within a repository and has not been accessed is considered orphaned."